### PR TITLE
gce: use latest available centos7 image

### LINKED ISSUE #1589: update default gce centos image
GCE has released an updated centos7 image: centos-7-v20160606

We should test and update our default.

### PR DESCRIPTION
**Pull request by ryane**
- [x] Installs cleanly on a fresh build of most recent master branch
- [x] Upgrades cleanly from the most recent release 
- [x] Updates documentation relevant to the changes (n/a)

* disk.sh -> gce-init.sh - removes disk resizing pkgs as this is not
  built-in to the image
* removes cloud-utils-growpart package as part of docker lvm setup

fixes #1589

### CODE REVIEW

File: terraform/gce/gce-init.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+set -e
+
+sudo yum -y install cloud-init cloud-initramfs-tools
+sudo reboot
```
Comment (ryane) [inline terraform/gce/gce-init.sh]: as of right now, I am not sure the cloud-init packages _need_ to be installed via terraform remote-exec nor if the reboot is required. But, perhaps this is more future proof should we do more with cloud-init going forward?

File: roles/docker/tasks/lvm.yml [hunk 2]
```diff
@@ -1,15 +1,4 @@
-
-# dependency for cloud-utils-growpart
-# which is required if we have GROWPART=yes (default) in /etc/sysconfig/docker-storage-setup
-- name: install cloud-utils-growpart package
```
Comment (ryane) [inline roles/docker/tasks/lvm.yml]: removal of this has only been tested on gce so far

**Comment by nigerianlegend** [comment]
@ryane Im running a test on this PR is that cool

**Comment by nigerianlegend** [comment]
@ryane
```
 TASK: [marathon | create consul acl for mantl-api] **************************** 
fatal: [lz-control-03] => One or more undefined variables: 'consul_acl_mantl_api_token' is undefined
FATAL: all hosts have already failed -- aborting
```

**Comment by ryane** [comment]
it looks like you need to re-run `./security-setup`. see #1525 which introduced the new variable

**Comment by nigerianlegend** [comment]
@ryane you were right everything is good

**Comment by ryane** [comment]
thanks @nigerianlegend. automated gce build also passed

**Pull request merged**